common/src/main/java/com/viaversion/viaversion/configuration/AbstractViaConfig.java

```diff
@@ -141,6 +145,15 @@ protected void loadFields() {
         ignoreLongChannelNames = getBoolean("ignore-long-1_16-channel-names", true);
         forcedUse1_17ResourcePack = getBoolean("forced-use-1_17-resource-pack", false);
         resourcePack1_17PromptMessage = getSerializedComponent("resource-pack-1_17-prompt");
+        map1_16WorldNames.clear();
+        JsonElement maybeMap = getSerializedComponent("map-1_16-world-names");
```

Comment:
This is not a component... Unsure if the Via config even supports subsections

Reply:
Is the actual shape of the config fine?

You are correct, the config item should already be a `Map<String, String>` which should be easily castable.